packages/paste-core/components/toast/src/types.ts

```diff
@@ -18,26 +26,33 @@ export interface ToastProps extends HTMLPasteProps<"div">, Pick<BoxProps, "eleme
   variant: ToastVariants;
   /**
    * Use this to set focus within the toast when it is rendered
+   *
+   * @default 'true'
```

Comment:
question: what use does including `@memberof` provide and should we include that line here too?

Reply:
there were a few other files that didn't use memberof but I just left the one comment here bc not entirely sure what value it provides

Reply:
I don't necessarily know if it gives us anything specifically, I get it automatically from a VS Code extension called Doc This.

Reply:
Reading up more, as we're typescript it likely gives us nothing.

Reply:
is Doc This the secret to doing these things quickly

Reply:
Potentially, I'm always looking for ways to make repetitive tasks quicker.